nixos/doc/manual/configuration/gpu-accel.xml

```diff
@@ -182,7 +182,11 @@ GPU1:
 	available for applications and lets them choose. A specific driver can
 	be forced as follows:
 
-	<programlisting><xref linkend="opt-hardware.opengl.extraPackages"/> = [
+	<programlisting><xref linkend="opt-hardware.opengl.extraPackages"/> = with pkgs; [
+  <package>amdvlk</package>
+];
```

Comment:
```suggestion
	<programlisting><xref linkend="opt-hardware.opengl.extraPackages"/> = [
  pkgs.<package>amdvlk</package>
];
```

Reply:
Using `with` is discouraged. See:

https://github.com/NixOS/nixpkgs/pull/96960